.env-dist

```diff
@@ -6,6 +6,8 @@ LOGOS_ORIGIN=
 COOKIE_SECRET=3895d33b5f9730f5eb2a2067fe0a690e298f55f5e382c032fd3656863412
 CSRF_SECRET=
 SESSION_DURATION_HOURS=48
+EMAIL_PREVIEW_ENABLED=
+EMAIL_RECIPIENT_DUMMY=localmonitor20200827@mailinator.com
```

Comment:
This is the same public Mailinator address we are already using for our [README](https://github.com/mozilla/blurts-server#trigger-breach-alert-email).

Reply:
For the previous version, QA mentioned how helpful it was to be able to send emails to specific addresses, not locked down to one (mailanator).  This easily allows testing on multiple email clients.  I wonder if it's possible to reintroduce this behavior?

If it's too difficult, we might at least default to sending to the current logged-in email.

Both of these options come with the auth caveat mentioned below.

Reply:
I added support for sending test emails to the primary email and Mailinator. It would not be hard to add additional emails. If OK I would like to break out additional enhancement into follow-up PRs so we can start testing the changes and see if they are working as expected.